### PR TITLE
Issue #472 Restore Kubernetes copyright notices

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
This reverts commit 6abc7d6b73aa903b8d542f311e72d528a7a27e32.

Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: hack/boilerplate/boilerplate.go.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (C) YEAR Red Hat, Inc.
+Copyright YEAR The Kubernetes Authors All rights reserved.
```
Comment (hferentschik) [inline hack/boilerplate/boilerplate.go.txt]: -1 This is not going to work. Now the prerelease task is failing when it verified the headers. All files which don't get the reverted header still have "Copyright (C) 2016 Red Hat, Inc.". 

_boilerplate.py_ needs to change to handle both variations of the header. I would suggest to use a regular expressions in this case which allows the two variants "The Kubernetes Authors All rights reserved" and "Red Hat, Inc."

Comment (hferentschik) [inline hack/boilerplate/boilerplate.go.txt]: This applies to all boilerplate templates.

Comment (LalatenduMohanty) [inline hack/boilerplate/boilerplate.go.txt]: @hferentschik I have missed it somehow.  Will fix it.

Comment (hferentschik) [inline hack/boilerplate/boilerplate.go.txt]: > I have missed it somehow. Will fix it.

👍

Comment (LalatenduMohanty) [inline hack/boilerplate/boilerplate.go.txt]: @hferentschik The easiest way to skip the files with Kubernetes boilerplate .But it is not right in my opinion as we might get new files from minikube in future. The current implementations is very deeply tied up with one boilerplate for one type of file.  I am still figuring out what would be the right thing to do here.

Comment (hferentschik) [inline hack/boilerplate/boilerplate.go.txt]: > The easiest way to skip the files with Kubernetes boilerplate 

-1

> The current implementations is very deeply tied up with one boilerplate for one type of file

I don't get it. There are boilerplate headers for the various file types, yes, but how is this all different than say the YEAR? Why don't you put COPYRIGHTHOLDER as a token in the various templates and treat this token the same way as the YEAR? See https://github.com/minishift/minishift/blob/8b500a691c670f5ee4de93a11b157de24f761fae/hack/boilerplate/boilerplate.py#L148-L148

Am I missing something here?

Comment (LalatenduMohanty) [inline hack/boilerplate/boilerplate.go.txt]: @hferentschik Never thought of putting the copyrightholder as a token as I thought it wont be elegant to put the whole paragraph as a token. But lets try that.

File: hack/boilerplate/boilerplate.go.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (C) YEAR Red Hat, Inc.
+COPYRIGHTHOLDER.
```
Comment (hferentschik) [inline hack/boilerplate/boilerplate.go.txt]: I was first going to ask why you ended up to replace the whole line with this token and not just the copy right holder itself. But I get it now. Looking at _boilerplate.py_, I have to say I find this a strange way of doing in general. But let's be done with it for now.

**Comment by hferentschik** [comment]
Merged, thanks.

**Pull request merged**